modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: REFERENCE
+
+[id="ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds_{context}"]
+= Integration between shared resources, Insights Operator, and OpenShift Builds
+
+[role="_abstract"]
+Integration between shared resources, Insights Operator, and OpenShift Builds makes using Red Hat subscriptions (RHEL entitlements) easier.
+
+Previously, in {product-title} 4.9.x and earlier, you manually imported your credentials or copied them to each project or namespace where you were running builds.
```

Comment:
create a link to the existing Builds documentation for this, like in 4.9, it is https://docs.openshift.com/container-platform/4.9/cicd/builds/running-entitled-builds.html

Reply:
Addressed in another related comment for this file.